db/migrate/20220108095456_create_cart_items.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateCartItems < ActiveRecord::Migration[6.0]
+  def change
+    create_table :cart_items do |t|
+      t.integer :quantity
+      t.references :product, null: false, foreign_key: true
+      t.references :cart, null: false, foreign_key: true
+      t.integer :total
+      t.integer :unit_price
```

Comment:
When and where we're calculating these values?

Reply:
I calculate these values in the models, but it is logic for cart task(DP2-8), I added this to it